src/components/views/messages/TextualBody.js

```diff
@@ -468,6 +374,7 @@ module.exports = React.createClass({
         return (
             <div
                 key="editedMarker" className="mx_EventTile_edited"
+                onClick={this._openHistoryDialog}
                 onMouseEnter={this._onMouseEnterEditedMarker}
                 onMouseLeave={this._onMouseLeaveEditedMarker}
             >{editedTooltip}<span>{`(${_t("edited")})`}</span></div>
```

Comment:
Zeplin suggests this should say something like "Edited at 12:00. Click to view edits." Unsure if we have the edit time here, but the click instruction should likely be added.

Reply:
It already says "Edited at 12:00" actually, will add the "Click to view edits." part though :+1: